packages/core/src/services/app/cashoutProvider.ts

```diff
@@ -67,20 +70,11 @@ export default class CashoutProviderService {
                     merchant.gps.longitude,
                     merchant.gps.latitude,
                 ]);
-                const dist = distance(userLocation, merchantLocation);
-                if (dist < parseInt(query.distance || '100', 10)) {
-                    merchant['distance'] = dist;
```

Comment:
Without this line, will the `sort` in the function return continue working?

Reply:
this distance was not being given back on the endpoint and it was also not needed on frontend, since the results should already be sorted. So no need

Reply:
but actually, it's not sorted, good point. Let me do that